modules/libcom/RTEMS/posix/rtems_init.c

```diff
@@ -1099,12 +1109,15 @@ POSIX_Init ( void *argument __attribute__((unused)))
     {
         unsigned int p;
         if (epicsThreadHighestPriorityLevelBelow(epicsThreadPriorityScanLow, &p)
-                                            == epicsThreadBooleanStatusSuccess)
+                                            != epicsThreadBooleanStatusSuccess)
         {
-            rtems_bsdnet_config.network_task_priority = p;
+            p = RTEMS_MAXIMUM_PRIORITY
+              - (RTEMS_MAXIMUM_PRIORITY - RTEMS_MINIMUM_PRIORITY) * epicsThreadPriorityScanLow / 100;
         }
+        printf(" This is network task prio (RTEMS) : %d , OSI Prio %d\n", p, epicsThreadPriorityScanLow);
```

Comment:
There are shell commands in RTEMS that can report this type of value for those times when you need to look.